src/main/java/me/zeroeightsix/kami/module/modules/render/BreakingESP.kt

```diff
@@ -21,29 +22,29 @@ import net.minecraftforge.fml.common.gameevent.TickEvent
 import org.kamiblue.event.listener.listener
 
 object BreakingESP : Module(
-    name = "BreakingESP",
-    description = "Highlights blocks being broken near you",
+    name = KamiLang.get("module.modules.render.BreakingESP.Breakingesp"),
+    description = KamiLang.get("module.modules.render.BreakingESP.HighlightsBlocksBeingBroken"),
```

Comment:
Use `ModuleName` and `ModuleDescription` for the name?

Reply:
That is the plan, I was just going to check that there wasn't any major complaints about the structuring of the file before going through and changing them all, currently I just have a janky python file which does it semi-automatically.